src/Workspaces/Core/Portable/FindSymbols/SymbolTree/SymbolTreeInfo_Source.cs

```diff
@@ -54,7 +54,7 @@ private static void FreeSymbolMap(MultiDictionary<string, ISymbol> symbolMap)
 
             // Order the documents by FilePath.  Default ordering in the RemoteWorkspace is
```

Comment:
I had 2 choices. where I make ordering always same or keep OOP and VS to compare its ordering and make them always same.

the first one was easier and simpler. 

since OOP incrementally build Workspace, document/project ordering might different than ordering in VS.